freight/notifiers/webhook.py

```diff
@@ -3,13 +3,19 @@
 from freight import http
 from freight.models import App, Task, User
 
-from .base import Notifier, generate_event_title
+from .base import Notifier, NotifierEvent, generate_event_title
 
 
 def stringify_date(date):
     return date.isoformat() + "Z" if date else None
 
 
+notifier_status = {
+    str(NotifierEvent.TASK_QUEUED): "queued",
```

Comment:
Why `str` it? I assume it'd be fine if you just then used `notifier_status.get(event, task.status_label)` right?

Reply:
Ya I'm bad

Reply:
@mattrobenolt updated: https://github.com/getsentry/freight/pull/231/commits/d78642804b2b23688983577952c0f671cde55399